cmd/speep/main.go

```diff
@@ -24,52 +24,81 @@ func main() {
+	getByNameShort := getCmd.String("n", "", "Find a shortcut with a name e.g. \"speep get -n Copy\"."+shortFlagName)
 
 	// peeper subcommand "add"
 	addCmd := flag.NewFlagSet("add", flag.ExitOnError)
 	// Flags(inputs) for "add" subcommand
-	addName := addCmd.String("name", "", "Name of the shortcut: Use \"\" for more than one word e.g. -name \"to the back of the line\"")
-	addKey := addCmd.String("key", "", "Key of the shortcut key\n e.g. \"speep add -name copy -shortcut Ctrl+C\"")
+	addName := addCmd.String("name", "", "Name of the shortcut: e.g. \"speep add --name copy --key Ctrl+C\"."+longFlagName)
+	addNameShort := addCmd.String("n", "", "Name of the shortcut: e.g. \"speep add -n copy -k Ctrl+C\"."+shortFlagName)
+	addKey := addCmd.String("key", "", "Key of the shortcut key\n e.g. \"speep add --name copy --key Ctrl+C\"")
+	addKeyShort := addCmd.String("k", "", "Key of the shortcut key\n e.g. \"speep add -n copy -k Ctrl+C\"")
 
 	// peeper subcommand "delete"
 	deleteCmd := flag.NewFlagSet("delete", flag.ExitOnError)
 	// Flags for "delete" subcommand
-	deleteAll := deleteCmd.Bool("all", false, "Delete whole shortcut list with \"speep delete -all\"")
-	deleteByName := deleteCmd.String("name", "", "Delete a shortcut by name e.g. \"speep delete -name Copy\"")
+	deleteAll := deleteCmd.Bool("all", false, "Delete whole shortcut list with \"speep delete --all\"")
+	deleteAllShort := deleteCmd.Bool("a", false, "Delete whole shortcut list with \"speep delete -a\"")
+	deleteByName := deleteCmd.String("name", "", "Delete a shortcut by name e.g. \"speep delete --name Copy\"."+longFlagName)
+	deleteByNameShort := deleteCmd.String("n", "", "Delete a shortcut by name e.g. \"speep delete -n Copy\"."+shortFlagName)
 
+	// Return usage if no command specified
 	if len(os.Args) < 2 {
-		return errors.New("Please check how to use the app by typing either \"speep get\", \"speep add\" or \"speep delete\" for more details.")
+		return fmt.Errorf("No command and flag specified. \n %s", usage)
 	}
 
-	switch os.Args[1] {
-	case "get":
-		return HandleGet(getCmd, getAll, getByName)
-	case "add":
-		return HandleAdd(addCmd, addName, addKey)
-	case "delete":
-		return HandleDelete(deleteCmd, deleteAll, deleteByName)
-	default: // When user chose command which doesn't exist
-		fmt.Println("Please check how to use the app by typing either \"speep get\", \"speep add\" or \"speep delete\" for more details.")
-
+	// Check user input command and proceed to each process
+	command := os.Args[1]
+	switch {
+	case command == "get":
+		return HandleGet(getCmd, getAll, getAllShort, getByName, getByNameShort)
+	case command == "add":
+		return HandleAdd(addCmd, addName, addNameShort, addKey, addKeyShort)
+	case command == "delete":
+		return HandleDelete(deleteCmd, deleteAll, deleteAllShort, deleteByName, deleteByNameShort)
+	case command == "help" || command == "--help" || command == "-h":
+		return Handlehelp(usage)
```

Comment:
Tip: Maybe it would be easier to use `flag.Usage`?
- [Example of how to set it](https://github.com/kinbiko/mokku/blob/main/cmd/mokku/mokku.go#L40)
- [Example of how to use it](https://github.com/kinbiko/mokku/blob/main/cmd/mokku/mokku.go#L32-L36)